parm/config/gfs/config.atmanl

```diff
@@ -5,12 +5,16 @@
 
 echo "BEGIN: config.atmanl"
 
-export CASE_ANL=${CASE}
-export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/config/
-export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/lists/gdas_prototype_3d.yaml
-export ATMVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/atm/variational/3dvar_drpcg.yaml
+if [[ ${DOHYBVAR} = "YES" ]]; then
+    # shellcheck disable=SC2153
+    export CASE_ANL=${CASE_ENS}
+else
+    export CASE_ANL=${CASE}
+fi
+export OBS_LIST="${HOMEgfs}/parm/gdas/atm/obs/lists/gdas_prototype_3d.yaml.j2"
```

Comment:
A recent PR into global-workflow `develop` (PR #2366) changed a number of top level variable/path settings. I am going to provide suggested changes that can be committed directly to help you incorporate related changes.

```suggestion
export OBS_LIST="${PARMgfs}/gdas/atm/obs/lists/gdas_prototype_3d.yaml.j2"
```

Reply:
Change committed at 36289eef